nexus/tests/integration_tests/projects.rs

```diff
@@ -245,32 +245,14 @@ async fn test_project_deletion_with_image(cptestctx: &ControlPlaneTestContext) {
         delete_project_expect_fail(&url, &client).await,
     );
 
-    // TODO: finish test once image delete is implemented. Image create works
-    // and project delete with image fails as expected, but image delete is not
-    // implemented yet, so we can't show that project delete works after image
-    // delete.
     let image_url = format!("/v1/images/{}", image.identity.id);
-    NexusRequest::expect_failure_with_body(
-        client,
-        StatusCode::INTERNAL_SERVER_ERROR,
-        Method::DELETE,
-        &image_url,
-        &image_create_params,
-    )
-    .authn_as(AuthnMode::PrivilegedUser)
-    .execute()
-    .await
-    .unwrap();
-
-    // TODO: delete the image
-    // NexusRequest::object_delete(&client, &image_url)
-    //     .authn_as(AuthnMode::PrivilegedUser)
-    //     .execute()
-    //     .await
-    //     .expect("failed to delete image");
+    NexusRequest::object_delete(&client, &image_url)
+        .authn_as(AuthnMode::PrivilegedUser)
+        .execute()
+        .await
+        .expect("failed to delete image");
```

Comment:
I would add a check here that an attempt to fetch the image now 404s

Reply:
done in bef3de54e